### DISCUSSION TITLE
What do I need to do to generate your source code into a single file that has a lot of similar language code and various libraries, including Ghelper.dll, even if I don't make changes to your source code

### DISCUSSION THREAD

**Discussion by Mi3sbbbi** [seerge/g-helper#1432] [General]
What do I need to do to generate your source code into a single file that has a lot of similar language code and various libraries, including Ghelper.dll, even if I don't make changes to your source code

**Comment by IceStormNG** [comment]
You mean how to build the single-file GHelper like that one offered here for download?

`dotnet publish app/GHelper.sln --configuration Release --runtime win-x64 -p:PublishSingleFile=true --no-self-contained` 
Run this through the VS Developer Shell. It will generate the single file distribution. You can find them in 
`app/bin/x64/Release/net7.0-windows/win-x64/publish/`

**Reply by Mi3sbbbi** [reply]
thank you very much for your help
